### ISSUE TITLE
[Screen Reader-Home Page-Blog] Name Property is not defined for the 'Print ' button showing on Blog Page.

### ISSUE THREAD

**Issue by v-mesala** [Microsoft/pxt#2623]
**User Experience:**
Users with Visual disabilities might not able to understand the controls if Name Property is not defined. Like in the scenario Name Property is not defined for the Print button. 

**Test Environment**
OS: RS2 Version 1703(OS Build 15063.2)
Platform: Edge.
Screen reader: Narrator

**Repro Steps**
1-Navigate to url:- https://makecode.com/
2-Navigate to the Blog control lying in the header section on the page and select it.
3-Navigate to the Print button showing on blog page.
4-Verify that Name Property is defined for the Print bbutton or not.

**Actual Result**
Name Property is not defined for the 'Print ' button showing on Blog Page.

**Expected Result**
Name Property should be defined for the 'Print ' button showing on Blog Page.

**MAS Reference**
https://microsoft.sharepoint.com/teams/msenable/Documents/LCA/MAS%20-%20July%201,%202017%20Release/MAS%204.1.2%20%E2%80%93%20Name,%20Role,%20Value.docx?d=w248054a65e6847719e1e242fb5025730&csf=1

**Suggested fix:**
1- Using aria-label to give an invisible name to element, where name/title property is not used 
e.g.  
'Close' button can have only ‘X’, to make it more meaningful aria-label attribute is used with ‘Close’ text to help AT’s without showing the content on UI 
 
<div id="box"> 
   This is a pop-up box. 
     <button aria-label="Close" onclick="document.getElementById('box').style.display='none';" class="closebutton"> X </button>                                          
</div> 
 
2-   Using ARIA role to expose the control type of user interface component 
e.g. 
<div role="toolbar" tabindex="0" id="customToolbar" >  
<img src="img/btn1.gif" role="button" tabindex="-1" alt="Home" id="b1" title="Home"> 
 
3-   Using ARIA attributes to expose the element’s state  
e.g. 
<div role="button" id="customToolbar" aria-checked=”” aria-disabled=”” aria-haspopup=”” aria-expanded=”” aria-selected=””> 

**Refer Attachment:**
![mas4 1 2_home page_blog](https://user-images.githubusercontent.com/30434417/28869616-f68701da-779a-11e7-840f-9199c0460a40.jpg)

**Comment by v-mesala** [comment]
Bug is verified on URL : https://makecode.com/ bug still exists .Hence reactivated the bug and added the Active tag.

**Comment by v-mesala** [comment]
Bug is verified on URL : https://makecode.com/ bug still exists .Hence reactivated the bug removed the fixed tag and added the Active tag.

**Comment by abchatra** [comment]
@veler reopened bug

**Comment by veler** [comment]
The issue is fixed but not deployed yet.

**Comment by v-mesala** [comment]
Verified on URL: https://makecode.com/ and issue is fixed now. Hence closing the bug
